### ISSUE TITLE
clazz?

### ISSUE THREAD

**Issue by zachary-foster** [ropensci/binomen#9]
Minor question. There are multiple occurrences of `clazz` in the `README.md`. Should this be "class", or are you trying to avoid using the reserved word "class"?

**Comment by sckott** [comment]
good catch, but yeah, trying to avoid `class` - most importantly i think cause of the fxn `class()` in base R, but also cause class is a reserved word in SQL databases (though that's not too relevant here)

**Comment by zachary-foster** [comment]
ahh, ok, that makes sense. Just thought I would check.

**Comment by sckott** [comment]
interested to hear how using this goes, haven't used it a whole lot IRL

**Comment by zachary-foster** [comment]
Ok, i will let you know once I play around with it a bit more. I want a taxonomy class system for my metacoder package. I already have a system I like that is not class-based. On first glance, it seems fundamentally different that what is implemented here, but I would not mind adapting to `binomen` if it can fit my needs. I would like to avoid spawning my own classes if others exist to avoid cluttering the R ecosystem if possible.
